src/components/item/publish/CCLicenseSelection.js

```diff
@@ -84,6 +82,16 @@ const CCLicenseSelection = ({ item }) => {
     setOpen(false);
   };
 
+  const handleRemove = (event) => {
+    event.preventDefault();
+    updateCCLicense({
+      id: itemId,
+      name: itemName,
+      settings: { ccLicenseAdaption: '' },
+    });
+    setOpen(false);
```

Comment:
I see it's actually your CCDialog component which controls the opening (since it has the button). So you should define the open state in CCDialog. instead of closing the dialog in `handleRemove`, do it in CCDialog: 
```js
const onClick = () => {
    handleSubmit()
    setOpen(false)
}
```